frontend/app/src/App.test.tsx

```diff
@@ -1527,10 +1528,33 @@ describe("handles HostCommunication messaging", () => {
       })
     )
 
-    expect(clearCacheFunc).toHaveBeenCalledWith()
+    expect(sendMessageFunc).toHaveBeenCalledWith({ clearCache: true })
```

Comment:
updating existing test; this test is a little more useful if mocks sendMessage and asserts that it was called with `{ clearCache: true }`